src/transforms/projectionpursuit.jl

```diff
@@ -0,0 +1,201 @@
+      I += ((2j+1)/2) * (mean(Pⱼ₋₁))^2
+    end
+  end
+  I
+end
+
+# projection index of the standard multivariate Gaussian
+function gindex(transform, N, q)
+  n  = transform.n
+  E  = 1.0 * I(q)
+  Iₙ = Matrix{Float64}(undef, n, q)
+  for i = 1:n
+    S = randn(N, q)
+    for j = 1:q
+      Iₙ[i,j] = pindex(transform, S, E[:,j])
+    end
+  end
+  Iₙ
+end
+
+function alphaguess(transform, Z)
+  q = size(Z, 2)
+  E = 1.0 * I(q)
+  
+  # objective function
+  func(α) = pindex(transform, Z, α)
+  
+  # evaluate score along axes
+  i  = argmax(func, eachcol(E))
+  α  = E[:,i]
+  Iₐ = func(α)
+  
+  # evaluate score along diagonals
+  diag(α, s, e) = (1/√(2+2s*α'*e)) * (α + s*e)
+  for eᵢ in eachcol(E)
+    d₊ = diag(α, +1, eᵢ)
+    d₋ = diag(α, -1, eᵢ)
+    f₊ = func(d₊)
+    f₋ = α'*eᵢ != 1.0 ? func(d₋) : 0.0
+    f, d = f₊ > f₋ ? (f₊, d₊) : (f₋, d₋)
+    if f > Iₐ
+      α  = d
+      Iₐ = f
+    end
+  end
+  
+  α
+end
+
+function neldermead(transform, Z, α₀)
+  f(α) = -pindex(transform, Z, α)
+  op = optimize(f, α₀)
+  minimizer(op)
+end
+
+function alphamax(transform, Z)
+  α = alphaguess(transform, Z)
+  neldermead(transform, Z, α)  
+end
+
+function rmstructure(transform, Z, α)
+  q = length(α)
+  Q, R = qr(hcat(α, rand(q, q-1)))
+  while norm(diag(R)) < transform.tol
+    Q, R = qr(hcat(α, rand(q, q-1)))
+  end
+  T = Z * Q
+  
+  # remove structure of first rotated axis
+  newtable, colcache = apply(Quantile(), Tables.table(T))
+  T[:,1] = Tables.getcolumn(newtable, 1)
+  
+  T * Q', Q, colcache
+end
+
+function stopcriteria(transform, Z, qgauss)
+  _, q = size(Z)
+  E = 1.0 * I(q)
+  Iₐ = [pindex(transform, Z, col) for col in eachcol(E)]
+  any(qgauss .< Iₐ)
+end
+
+function applyfeat(transform::ProjectionPursuit, table, prep) 
+  cols = Tables.columns(table)
+  names = Tables.columnnames(cols)
+
+  trans = Quantile() → EigenAnalysis(:VDV)
+  ttable, tcache = apply(trans, table)
+
+  # standard Gaussian scores and quantiles
+  Z = Tables.matrix(ttable)
+  N, q = size(Z)
+  Igauss = gindex(transform, N, q) 
+  qgauss = quantile.(eachcol(Igauss), transform.perc)
+  
+  iter = 0
+  cache = Tuple[]
+  while stopcriteria(transform, Z, qgauss)
+    α = alphamax(transform, Z)
+    Z, Q, colcache = rmstructure(transform, Z, α)
+    iter += 1
+    if iter ≥ transform.maxiter
+      @warn "Maximum number of iterations ($count) reached."
+      break
+    end
+    push!(cache, (Q, colcache))
+  end
+
+  𝒯 = (; zip(names, eachcol(Z))...)
+  newtable = 𝒯 |> Tables.materializer(table)
+  newtable, (cache, tcache)
+end
+
+function revertfeat(::ProjectionPursuit, newtable, fcache)
+  cols = Tables.columns(newtable)
+  names = Tables.columnnames(cols)
+  ppcache, tcache = fcache
+
+  t = newtable
+  for (Q, colcache) in reverse(ppcache)
+    # rotate the data 
+    T = Tables.matrix(t) * Q
+
+    #Convert to table
+    Tt = Tables.table(T)
+
+    #Inverting the transformation T
+    Q⁻¹Tt  = revert(Quantile(), Tt, colcache)
```

Comment:
```suggestion
    Q⁻¹Tt  = revert(Quantile(1), Tt, colcache)
```
Same here, please double check